example/wasm/README.md

```diff
@@ -0,0 +1,30 @@
+# wasm (dart)
+
+A simple demo for [ml_algo](https://pub.dev/packages/ml_algo)
+Decision tree example for predicting an iris' species based on its dimensions.
+This simple js web demo uses Dart Web Assembly support
+
+## building/running
+- Serve the output, example
+
+$ cd site
+$ dart pub global activate dhttpd
+$ dart pub global run dhttpd
+
+- Open your brwoser at : 
+
+http://localhost:8080/
```

Comment:
@flutter-painter could you  please correct the path here? It seems to me that the actual path is `http://localhost:8080/site/index.html` (at least I was able to reach the demo by this address)